packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesDevLauncherController.java

```diff
@@ -0,0 +1,116 @@
+package expo.modules.updates;
+
+import android.content.Context;
+
+import androidx.annotation.Nullable;
+
+import org.json.JSONObject;
+
+import java.util.HashMap;
+
+import expo.modules.updates.db.DatabaseHolder;
+import expo.modules.updates.db.entity.AssetEntity;
+import expo.modules.updates.db.entity.UpdateEntity;
+import expo.modules.updates.launcher.DatabaseLauncher;
+import expo.modules.updates.launcher.Launcher;
+import expo.modules.updates.loader.RemoteLoader;
+import expo.modules.updates.manifest.Manifest;
+import expo.modules.updates.selectionpolicy.ReaperSelectionPolicyDevelopmentClient;
+import expo.modules.updates.selectionpolicy.SelectionPolicy;
+import expo.modules.updatesinterface.UpdatesInterface;
+
+public class UpdatesDevLauncherController implements UpdatesInterface {
+
+  private JSONObject mRawManifest;
+
+  public UpdatesDevLauncherController(Context context) {
+    UpdatesController.initializeInternal(context);
+    setDevelopmentSelectionPolicy();
```

Comment:
`fetchUpdateWithConfiguration` runs the loader code. we don't want to do it from the constructor here.

Reply:
This new class delegates control of timing (i.e. when to load the update) to the calling class (the dev launcher) by having a separate method, rather than assuming, as UpdatesController does, that it should start loading the update as soon as it's initialized.